src/pages/AddMembersPage/NewMemberEntry.tsx

```diff
@@ -29,6 +88,7 @@ const NewMemberEntry = ({
             fullWidth
             error={error?.name ? true : undefined}
             autoComplete={'off'}
+            disabled={isFetched}
```

Comment:
![image](https://github.com/coordinape/coordinape/assets/83605543/0ea6b0ac-e162-4967-9951-54ee6409046b)

the field has state disabled, but it looks exactly the same as the other Name fields, which is confusing. I think we should make it disabled and have a grey background so its clear that field is not editable @wingfeatherwave

Reply:
fixed